### PR TITLE
feat: Explaining what the project is all about.

### LINKED ISSUE #48: [FEATURE] <Explain this project on the "production site">
### Description

We deploy this site to Vercel [here](https://defaang.vercel.app/), but it's just a starter template, and it doesn't say anything about the project.

Some people might be confused about that: #45

It'd be good to explain what this project is about there, even if it's a temporary thing.

### PR DESCRIPTION
**Pull request by Jonyango**
### 🛠️ Fixes Issue

Closes #48 

### ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots
desciption

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -6,79 +6,22 @@ const Home: NextPage = () => {
...
 
-      <main className="flex w-full flex-1 flex-col items-center justify-center px-20 text-center">
-        <h1 className="text-6xl font-bold">
-          Welcome to{' '}
-          <a className="text-blue-600" href="https://nextjs.org">
-            Next.js!
-          </a>
+      <main className="w-full  px-20 text-center">
+        <h1 className="text-5xl font-medium">
+        Hello There!  Welcome to Defaang. <br />
```
Comment (ykdojo) [inline pages/index.tsx:14]: Can we change Defaang to defaang? I think the lowercased brand is going to be a bit more friendly

**Review by ykdojo — CHANGES_REQUESTED** [review]

**Comment by ykdojo** [comment]
Thank you so much! I added one comment

File: pages/index.tsx (hunk 1 above)
Comment (Jonyango) [reply pages/index.tsx:14]: Okay

**Comment by Jonyango** [comment]
> Thank you so much! I added one comment

I have applied the requested changes.

**Comment by Jonyango** [comment]
I realised that I couldn't raise 2 PRs from the same branch if one hasn't been closed(Once I had pushed the other changes). So I will add the second issue that I worked on the same PR. I hope That is okay.

**Pull request merged**